src/core/common/instance.cpp

```diff
@@ -167,6 +167,14 @@ void Instance::Reset(void)
     otPlatReset(this);
 }
 
+#if OPENTHREAD_RADIO
+void Instance::RadioStackReset(void)
```

Comment:
How about `ResetRadioStack()` insetad? 
(though not required by our style guide, we often start a method name with the "verb").

Reply:
Done.